__tests__/data/am_input.json

```diff
@@ -664,6 +664,137 @@
       }
     }
   },
+  {
+    "message": {
+      "type": "track",
+      "event": "Sample track event BEFORE IDENTIFY1**",
+      "sentAt": "2020-09-17T15:07:13.171Z",
+      "userId": "0572f78fa49c648e",
+      "channel": "mobile",
+      "context": {
+        "os": {
+          "name": "Android",
+          "version": "9"
+        },
+        "app": {
+          "name": "AMTestProject",
+          "build": "1",
+          "version": "1.0",
+          "namespace": "com.rudderstack.android.rudderstack.sampleAndroidApp"
+        },
+        "device": {
+          "id": "0572f78fa49c648e",
+          "name": "generic_x86_arm",
+          "type": "Android",
+          "model": "AOSP on IA Emulator",
+          "manufacturer": "Google",
+          "adid": "44c97318-9040-4361-8bc7-4eb30f665ca8"
```

Comment:
The spec'd property is `advertisingId`. It holds the value for `IDFA` for iOS and `ADID` for Android. Also, there is one more field - `adTrackingEnabled` which can tell you quickly whether the value in `advertisingId` is valid or not.

Reply:
The spec'd properties are `advertisingId` and `adTrackingEnabled` added.